runtime/README.md

```diff
@@ -2402,6 +2406,36 @@ The `exported_cdi` can be created by calling `DeriveContext` with the `export-cd
 The `exported_cdi_handle` is no longer usable after calling `REVOKE_EXPORTED_CDI_HANDLE` with it. After the `exported_cdi_handle`
 has been revoked, a new exported CDI can be created by calling `DeriveContext` with the `export-cdi` and `create-certificate` flags.
 
+### EXTERNAL_MAILBOX_CMD
+
+Command Code: `0x4558_544D` ("EXTM")
+
+**Note**: This command is only available in subsystem mode in 2.1+.
+
+Executes a mailbox command located at an AXI address.
+This allows for executing mailbox commands that are larger than the mailbox allows.
+
+This is currently mostly useful for FIRMWARE_LOAD (as part of an update) or SET_AUTH_MANIFEST.
```

Comment:
nit: I wonder if we can make this simpler by just adding a FIRMWARE_LOAD_EXT command. Or do you have other future use-cases in mind?

Reply:
We also need it for SET_AUTH_MANIFEST. Interestingly enough, it also might be more performant to use external commands since they are DMA'd into DCCM.